client/src/slices/credentials/credentialsSlice.ts

```diff
@@ -87,9 +92,15 @@ const credentialSlice = createSlice({
         state.credentials = []
         state.isLoading = false
       })
+      .addCase('demo/resetState', (state) => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const protocolVersion = state.protocolVersion
+        state = initialState
+        state.protocolVersion = protocolVersion
```

Comment:
Same here, I think we can do it like this instead:

```suggestion
        return {
            ...initialState,
            protocolVersion: state.protocolVersion
       }
```